test/rekt/features/pingsource/features.go

```diff
@@ -138,6 +138,9 @@ func SendsEventsWithEventTypes() *feature.Feature {
 
 	f := new(feature.Feature)
 
+	// Enable the eventtype auto-creation in configmap
+	f.Setup("enable eventtype auto-creation", eventtype.ApplyEventTypeConfigMap())
+
```

Comment:
I'm not sure we should be enabling experimental features in the rekt (non-experimental) tests. Maybe if the events are not created by the ping source it makes more sense to remove the assertion rather than enabling this?